### ISSUE TITLE
istio proxy access log format config not working

### ISSUE THREAD

**Issue by zhqworld** [istio/istio#11394]
**Describe the bug**
The latest istio snapshot build: 1.1.0-snapshot.5 allows to customize proxy access log by specifying access log format: https://github.com/istio/istio/blob/1.1.0-snapshot.5/install/kubernetes/helm/istio/templates/configmap.yaml#L25

I tried to set values in the "istio" configmap as following, but it didn't work.
```
accessLogFormat: '[%START_TIME%] %REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\n'
```

**Expected behavior**
envoy proxy should write log with new format.

**Steps to reproduce the bug**
1. change configmap of "istio" with above entry.
2. restart pilot pod.
3. create a new istio injected pod, check istio-proxy log. (no access log flowing while firing request)
4. the new injected istio-proxy container has following config:
```            {
             "name": "envoy.file_access_log",
             "config": {
              "format": "[%START_TIME%] %REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL% ",
              "path": "/dev/stdout"
             }
            }
           ] ```

Note; new line "\n" is missing in the runtime config dump of proxy.

And based on https://github.com/envoyproxy/envoy/issues/5193, seems for text formatted log, it needs "\n" in format string. So I guess istio config lost it somewhere between pilot and proxy.

**Comment by mathspanda** [comment]
You can try with `accessLogFormat: "[%START_TIME%] %REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\\n"`.

**Comment by zhqworld** [comment]
"\\n" not working either.

**Comment by duderino** [comment]
@GregHanson considering you did https://github.com/istio/istio/issues/10056 could you take this one on?

**Comment by linsun** [comment]
@GregHanson pointed me to https://github.com/istio/istio/pull/11449

**Comment by GregHanson** [comment]
resolved by PR #11449
